### DISCUSSION TITLE
Man kan ikke skrive i afsnittet “Kort beskrivelse” i arrangørpræsentationen

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#30] [Bugs] [answered]
Man kan slette men ikke skrive nyt. Her ved Roots&Jazz hos kulturens frivillige i København - som nu er en del af Copenhagen azz Festival:

 ![image](https://github.com/xooom-community/product/assets/167326849/4c4d7a08-a67b-4522-8c81-f03a2628558e)

**Comment by MartinQwist** [comment]
@mikkelsl Kan I kigge på denne? Den er ret kritisk. Jeg har meldinger fra andre, der ikke kan rette titlen på en arrangør.

**Reply by mikkelsl** [reply, marked as answer]
Har smidt den som opgave i Development: https://github.com/xooom-community/product/issues/203

**Comment by henrikbak** [comment]
@MartinQwist Feltet "Kort beskrivelse" er på et tidspunkt blevet begrænset til 80 karakterer. Vurderer ud fra det vedhæftede screenshot kunne det se ud til at denne arrangør er blevet oprettet, inden denne grænse blev indført, og derfor har man fået lov til at indtaste en beskrivelse der er væsentligt længere.

Løsningen er at lave en kort beskrivelse på maks. 80 karakterer i "Kort beskrivelse" og så lave den lange beskrivelse af arrangøren under "Om os" som findes på samme side.
